repos/Directory.Build.targets

```diff
@@ -86,6 +88,17 @@
     <WriteLinesToFile File="$(RepoCompletedSemaphorePath)ApplyPatches.complete" Overwrite="true" />
   </Target>
 
+  <Target Name="UpdateBuildToolFramework"
+    Condition="'$(EngCommonToolsShFile)' != ''"
+    Outputs="$(RepoCompletedSemaphorePath)UpdateBuildToolFramework.complete" >
```

Comment:
Style nit: align with `Name`. This isn't consistent, just what the style happens to be. 😛 (If `Target` got renamed as often as MSBuild tasks did, I'd be pushing to get rid of alignment here too.)